tests/agent/test_session_persister.py

```
@@ -0,0 +1,939 @@
+            messages, user_query, completed
+        )
+        assert public_result == private_result
+
+    def test_public_method_basic_output_structure(self, persister):
+        """Public method should produce valid ShareGPT-style trajectory."""
+        messages = [
+            {"role": "user", "content": "What is 2+2?"},
+            {"role": "assistant", "content": "4"},
+        ]
+        result = persister.convert_to_trajectory_format(
+            messages, "What is 2+2?", True
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)